### PR TITLE
Add Studio guide docs with plugin

### PR DESCRIPTION
**Pull request by kenjinp**
Adds a quick tutorial of how to show a studio of neuron images with a simple image plugin

Screenshot 2020-02-25 at 14 38 03

### CODE REVIEW

File: src/main/paradox/docs/nexus-web/studio/creating-a-studio.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+And now here's our dashboard. Now we can share this URL with other people and they'll be able to see the results of our query as well.
+
+![Neuron Images Dashboard](../assets/studio-guide-neurons-dashboard.png);
+
+## Adding a plugin
+
+But there's one problem, clicking on one of the file names will show us an error `No Plugins Configured`.
+
+Let's develop a small plugin the displays the image with a title. Here's an implementation in Typescript:
```
Comment (Stafeeva) [inline src/main/paradox/docs/nexus-web/studio/creating-a-studio.md]: it should be `a small plugin that displays`

File: src/main/paradox/docs/nexus-web/studio/creating-a-studio.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+};
+
+export default ImagePlugin;
+```
+
+> A simple plugin that fetches a file from the resource metadata and appends it as an image on the document
+
+## Uploading a plugin
+
+Adding this the bundled Javascript in the `/plugins` folder of the root of your nexas web instance, such as `/plugins/nexus-image-plugin/index.js`, will allow Nexus Web to find the file and display it. Be sure to restart your nexus-web instance.
```
Comment (Stafeeva) [inline src/main/paradox/docs/nexus-web/studio/creating-a-studio.md]: this or the? :)

Comment (Stafeeva) [reply src/main/paradox/docs/nexus-web/studio/creating-a-studio.md]: Nexas 🙀

Comment (kenjinp) [reply src/main/paradox/docs/nexus-web/studio/creating-a-studio.md]: Nexas from Texas

**Pull request merged**